examples/python/sqlc.json

```diff
@@ -8,7 +8,9 @@
       "gen": {
         "python": {
           "out": "src/authors",
-          "package": "authors"
+          "package": "authors",
+          "emit_sync_querier": true,
+          "emit_async_querier": true
```

Comment:
The "emit_*" style was something I didn't think much about when I first released sqlc. Feel free to pick different names if you'd like.

Reply:
The "emit_*" style seemed fine to me, nothing really better is coming to mind and it sticks with the style of other options.